src/idsk/components/footer-extended/footer-extended.js

```diff
@@ -79,6 +82,18 @@ FooterExtended.prototype.handleSubmitButtonClick = function (e) {
     toggleClass($infoQuestion, 'idsk-footer-extended-heart');
     toggleClass($heartSymbol, 'idsk-footer-extended-heart-visible');
     toggleClass($feedbackQuestion, 'idsk-footer-extended-display-none');
+
+    var $selection = this.$module.querySelector('#sort');
+    var $issueTextArea = this.$module.querySelector('#with-hint');
+
+    var selectedOption = $selection.value;
+    var issueText = $issueTextArea.value;
+
+    var email = 'idsk@vicepremier.gov.sk';
```

Comment:
tento email nemoze byt natvrdo v JS.. to by kazda paticka posielala mail na tu adresu